### DISCUSSION TITLE
is ESP32-C3 Super Mini supported?

### DISCUSSION THREAD

**Discussion by menotuu** [JAndrassy/ArduinoOTA#235] [Q&A] [answered]
is a ESP32-C3 Super Mini supported?

ESP32-C3FN4 RISC-V 32-Bit-Single-Core

https://www.google.com/search?q=ESP32-C3+Super+Mini

I am a non profi, so I dont know if [esp32 support](https://github.com/jandrassy/ArduinoOTA#esp8266-and-esp32-support) means, it support strictly all esp32 derivatives.

**Comment by JAndrassy** [comment, marked as answer]
it the Update library in esp32 core supports. 
why do you wan to use this library for the esp32. esp32 core has its own ArduinoOTA library?

**Comment by menotuu** [comment]
hm because I have no idea about that. I came via a youtube video here to ArduinoOTA.
